README.md

```diff
@@ -253,6 +253,8 @@ RTE_SDK. For example:
     sudo dpkg -i protobuf-c-compiler_0.15-1build1_amd64.deb
     ```
 
+__ATTENTION: Since Ubuntu trusty (14) is been dropped you can't download this packets anymore, although you can run the script in the code root `dep_install.sh` which will provide a full installation of protobuf2 untill we will support protobuf3.__
```

Comment:
I would limit the line size to 80 columns like the rest of this document (except links).